controller/recurring_job_controller.go

```diff
@@ -220,6 +224,51 @@ func (control *RecurringJobController) syncRecurringJob(key string) (err error)
 	return nil
 }
 
+func (control *RecurringJobController) cleanupVolumeRecurringJob(recurringJob *longhorn.RecurringJob) error {
+	// check if other recurring jobs contains same groups
```

Comment:
// Check if each group of the RJ contains other RJs. If No, it means the RJ is the last job of the group then Longhorn will clean up this group labels for all volumes.

Reply:
Addressed.